kernel/arch/dreamcast/sound/snd_stream.c

```diff
@@ -148,16 +151,13 @@ static void process_filters(snd_stream_hnd_t hnd, void **buffer, int *samplecnt)
     }
 }
 
-
-/* Performs stereo seperation for the two channels; this routine
-   has been optimized for the SH-4. */
 static void sep_data(void *buffer, int len, int stereo) {
     register int16  *bufsrc, *bufdst;
     register int    x, y, cnt;
 
     if(stereo) {
         bufsrc = (int16*)buffer;
-        bufdst = sep_buffer[0];
+        bufdst = (int16*)sep_buffer[0];
```

Comment:
```suggestion
        bufdst = (int16 *)sep_buffer[0];
```
@ljsebald always makes me change that. :)

Reply:
I always write like this by the way. But here it was different; I followed the style of the environment.